artifactory-enterprise/azuredeploy.json

```diff
@@ -0,0 +1,863 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmSku": {
+      "type": "string",
+      "defaultValue": "Standard_A4",
+      "metadata": {
+        "description": "Size of VMs in the VM Scale Set."
+      }
+    },
+    "clusterName": {
```

Comment:
add a unique defaultValue

Reply:
https://github.com/Azure/azure-quickstart-templates/blob/6a3bcadb6788ce5a39e60b11570ea2a47087ca43/artifactory-enterprise/azuredeploy.parameters.json#L8

Reply:
On this parameter, you need to add a defaultValue property.  The value that property must use the uniqueString() function with a valid see to generate a unique name